Error window creation fails if the error text contains htm

An API is returning the following error:
```
odata.ODataException: ODataError(code=Request-URI Too Long(414), message=<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN""http://www.w3.org/TR/html4/strict.dtd">
<HTML><HEAD><TITLE>Request URL Too Long</TITLE>
<META HTTP-EQUIV="Content-Type" Content="text/html; charset=us-ascii"></HEAD>
<BODY><h2>Request URL Too Long</h2>
<hr><p>HTTP Error 414. The request URL is too long.</p>
</BODY></HTML>
```

Which results in:
```
Caused by: java.lang.IllegalArgumentException: HTML must contain exactly one top level element (ignoring text nodes). Found 2 elements with the tag names span, html
	at com.vaadin.flow.component.Html.setOuterHtml(Html.java:145) ~[flow-server-23.3.17.jar:23.3.17]
	at com.vaadin.flow.component.Html.<init>(Html.java:113) ~[flow-server-23.3.17.jar:23.3.17]
	at com.flowingcode.vaadin.addons.errorwindow.ErrorWindow.createErrorLabel(ErrorWindow.java:259) ~[error-window-vaadin-3.5.1.jar:3.5.1]
	at com.flowingcode.vaadin.addons.errorwindow.ErrorWindow.createMainLayout(ErrorWindow.java:165) ~[error-window-vaadin-3.5.1.jar:3.5.1]
	at com.flowingcode.vaadin.addons.errorwindow.ErrorWindow.initWindow(ErrorWindow.java:145) ~[error-window-vaadin-3.5.1.jar:3.5.1]
	at com.flowingcode.vaadin.addons.errorwindow.ErrorWindow.lambda$new$26ff2c46$1(ErrorWindow.java:107) ~[error-window-vaadin-3.5.1.jar:3.5.1]
	at com.vaadin.flow.component.ComponentEventBus.fireEventForListener(ComponentEventBus.java:233) ~[flow-server-23.3.17.jar:23.3.17]
	at com.vaadin.flow.component.ComponentEventBus.fireEvent(ComponentEventBus.java:222) ~[flow-server-23.3.17.jar:23.3.17]
	at com.vaadin.flow.component.Component.fireEvent(Component.java:393) ~[flow-server-23.3.17.jar:23.3.17]
	at com.vaadin.flow.component.ComponentUtil.fireEvent(ComponentUtil.java:416) ~[flow-server-23.3.17.jar:23.3.17]
	at com.vaadin.flow.component.ComponentUtil.onComponentAttach(ComponentUtil.java:233) ~[flow-server-23.3.17.jar:23.3.17]
	at com.vaadin.flow.internal.nodefeature.ComponentMapping.lambda$onAttach$0(ComponentMapping.java:104) ~[flow-server-23.3.17.jar:23.3.17]
	at java.util.Optional.ifPresent(Optional.java:178) ~[?:?]
	at com.vaadin.flow.internal.nodefeature.ComponentMapping.onAttach(ComponentMapping.java:103) ~[flow-server-23.3.17.jar:23.3.17]
	at com.vaadin.flow.internal.StateNode.lambda$fireAttachListeners$8(StateNode.java:858) ~[flow-server-23.3.17.jar:23.3.17]
```

I don't understand why `Span` isn't used directly here:
https://github.com/FlowingCode/ErrorWindowAddon/blob/26de67daea91d01b9f41b570fd5c701d7a74b98c/src/main/java/com/flowingcode/vaadin/addons/errorwindow/ErrorWindow.java#L313-L315

--- a/pom.xml
+++ b/pom.xml
@@ -3,7 +3,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.flowingcode.vaadin.addons</groupId>
 	<artifactId>error-window-vaadin</artifactId>
-	<version>3.7.1-SNAPSHOT</version>
+	<version>4.0.0-SNAPSHOT</version>
 	<name>Error Window Add-on</name>
 	<description>Error handler that displays a dialog with exception information</description>
 	<url>https://www.flowingcode.com/en/open-source/</url>

--- a/src/main/java/com/flowingcode/vaadin/addons/errorwindow/ErrorWindow.java
+++ b/src/main/java/com/flowingcode/vaadin/addons/errorwindow/ErrorWindow.java
@@ -23,12 +23,14 @@ package com.flowingcode.vaadin.addons.errorwindow;
 
 import com.vaadin.flow.component.Component;
 import com.vaadin.flow.component.Html;
+import com.vaadin.flow.component.Text;
 import com.vaadin.flow.component.UI;
 import com.vaadin.flow.component.button.Button;
 import com.vaadin.flow.component.button.ButtonVariant;
 import com.vaadin.flow.component.dependency.CssImport;
 import com.vaadin.flow.component.dialog.Dialog;
 import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
 import com.vaadin.flow.component.icon.VaadinIcon;
 import com.vaadin.flow.component.orderedlayout.FlexComponent.Alignment;
 import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
@@ -216,7 +218,7 @@ public class ErrorWindow extends Dialog {
     title.getElement().getStyle().set("width", "100%");
     mainLayout.add(title);
 
-    final Html errorLabel = createErrorLabel();
+    Component errorLabel = createErrorLabel();
     mainLayout.add(errorLabel);
     mainLayout.setHorizontalComponentAlignment(Alignment.START, errorLabel);
 
@@ -300,18 +302,21 @@ public class ErrorWindow extends Dialog {
     pw.flush();
     return baos.toString();
   }
-  
-  protected Html createErrorLabel() {
-    String label = errorMessage == null ? i18n.getDefaultErrorMessage() : errorMessage;
+
+  protected Component createErrorLabel() {
+    Div errorLabel = new Div();
+    errorLabel.setClassName("errorlabel");
+
     if (productionMode) {
-      label =
-          label.concat(
-              String.format(
-                  "<br />%s<br /><span class='uuid'>%s</span>",
-                  i18n.getInstructions(), uuid));
+      Div instructions = new Div(new Text(i18n.getInstructions()));
+      Span uuidSpan = new Span(uuid);
+      uuidSpan.setClassName("uuid");
+      errorLabel.add(instructions, uuidSpan);
+    } else {
+      String label = errorMessage == null ? i18n.getDefaultErrorMessage() : errorMessage;
+      errorLabel.setText(label);
     }
-    final Html errorLabel = new Html("<span>" + label + "</span>");
-    errorLabel.getElement().getClassList().add("errorlabel");
+
     return errorLabel;
   }
 }

